### ISSUE TITLE
Overfitting but increasing ROUGE scores

### ISSUE THREAD

**Issue by brn887** [tensorflow/tensor2tensor#1796]
Hello, I'm working on summarization and I have a small dataset (around 35k train, 2k validation and 2k test). I used TokenTextEncoder with the following parameters:

--model=lstm_seq2seq_attention_bidirectional_encoder
--hparams_set=lstm_bahdanau_attention
--hparams='batch_size=1024'
--train_steps=50000
--local_eval_frequency=1000
--eval_steps=100
--eval_throttle_seconds=10

The model starts overfitting after around 10k, but the ROUGE scores keep increasing on the validation data. Any idea or suggestion for this?

Thanks!

<img width="352" alt="loss" src="https://user-images.githubusercontent.com/61744776/75964888-04394e00-5ed9-11ea-822f-db8263a54807.png">

<img width="348" alt="rouge" src="https://user-images.githubusercontent.com/61744776/75964915-13200080-5ed9-11ea-879e-882206c5205b.png">
